### ISSUE TITLE
It is not possible to slice regressions for a single CL because of autobisect

### ISSUE THREAD

**Issue by natorion** [catapult-project/catapult#3372]
I was triaging the alerts associated with V8's switch to I+TF. I wanted to cut out the execution performance regressions from https://bugs.chromium.org/p/chromium/issues/detail?id=700364 into separate bugs for easier handling and to set them to "WontFix":

1.) Open alerts for https://bugs.chromium.org/p/chromium/issues/detail?id=700364
2.) Select alerts associated in https://bugs.chromium.org/p/chromium/issues/detail?id=700820 and create a new bug for them =>  https://bugs.chromium.org/p/chromium/issues/detail?id=700820 created.
3.) A new bug automatically triggers and autobisect
4.) Bisect finishes and marks the bug as duplicate: https://bugs.chromium.org/p/chromium/issues/detail?id=700820#c8
5.) ¯\_(ツ)_/¯

**Comment by anniesullie** [comment]
We've had requests for a user-added label that would basically tell bisect "do not dupe this bug into another one", would that solve the issue? @simonhatch

**Comment by natorion** [comment]
Yes. As I make that decision during bug creation, a checkbox in the chromeperf popup for bug creation would make sense. If it simply adds a label which prevents duplication marking, that would be perfect.
